synapse/storage/background_updates.py

```diff
@@ -158,11 +168,16 @@ def __init__(self, hs: "HomeServer", database: "DatabasePool"):
 
         self._background_update_performance: Dict[str, BackgroundUpdatePerformance] = {}
         self._background_update_handlers: Dict[str, _BackgroundUpdateHandler] = {}
+        # TODO: all these bool flags make me feel icky---can we combine into a status
```

Comment:
Do you see any trouble with doing so by using the enum you are creating ? It seems a small enough change to be included in this PR.

Reply:
I just didn't want to dive into the machinery behind the background updater here---this was meant to be quick and dirty to get a useful metric out.